### ISSUE TITLE
core: frontend: multi-language support

### ISSUE THREAD

**Issue by ES-Alexander** [bluerobotics/BlueOS#1937]
### Current behaviour

Everything is in English. Many people are more comfortable with other languages, and could benefit from being able to view the interface in their preferred language.

This is particularly relevant because vehicle access is often without an internet connection, so it may not always be possible to use automatic (cloud-based) browser translation features.

### Expected or desired behaviour

Ideally users should be able to select their preferred language for the visual elements, and developers should be able to contribute translation files (perhaps even as a plugin / sub-extension, so they can be dynamically installed for people who want them).

A full implementation of this would likely require a significant refactor of the codebase, and I'm not sure how that can best be approached, or how it could be made to include support for extensions (they may need to be handled separately, although hopefully it will at least be possible to use a consistent system). That said, it's not critical to implement everything at once - it would likely be quite helpful to start with even just the button labels, and we could prioritise standard pages over pirate-mode only ones.

### Prerequisites

- [X] I have checked to make sure that a similar request has not already been filed or fixed.
